elements/pl-barcode-scan/pl-barcode-scan.js

```diff
@@ -0,0 +1,7 @@
+/* global window, crc16 */
+
+() => {
+  window.crc16 = (base36) => {
+    return crc16(base36);
+  };
+};
```

Comment:
This is a little weird - why is this necessary? And AFAICT, this function isn't ever invoked, so I don't think it actually does anything.

Reply:
Seems to work without it. The example had this pattern. File eliminated.

Reply:
Resolved in e6feb8e.